### ISSUE TITLE
[0.11.0-rc1] Cluster with -join need ALL node to restart

### ISSUE THREAD

**Issue by PierreF** [influxdata/influxdb#6027]
I've seen #5602, it's great! With the addition of #5706 it now possible to start cluster on every node with just:
```
$ influxd -join node1:8091,node2:8091,node3:8091 -hostname $HOSTNAME
```

As I would expect (and from my understanding of PR #5602), to restart node I could still keep the SAME command line; e.g. : 
```
$ influxd -join node1:8091,node2:8091,node3:8091 -hostname $HOSTNAME
```

This works well only if at least one node was still alive. If all node are down (let's say a power outage), if only two node are back online (let's say of node3 had an hardware fault following the power outage), I got the following:
```
2016/03/16 14:26:02 InfluxDB starting, version 0.11.0rc1, branch 0.11, commit 441772e87782c27a679043071f7181f7928bfbb2
2016/03/16 14:26:02 Go version go1.4.3, GOMAXPROCS set to 4
2016/03/16 14:26:02 no configuration provided, using default settings
[meta] 2016/03/16 14:26:02 Starting meta service
[meta] 2016/03/16 14:26:02 Listening on HTTP: [::]:8091
[metastore] 2016/03/16 14:26:02 Using data dir: /root/.influxdb/meta
[metastore] 2016/03/16 14:26:02 Waiting for 2 join peers.  Have [node1:8088]. Asking nodes: [node1:8091 node2:8091 node3:8091]
[metastore] 2016/03/16 14:26:03 Waiting for 1 join peers.  Have [node1:8088 node2:8088]. Asking nodes: [node1:8091 node2:8091 node3:8091]
[...]
[metastore] 2016/03/16 14:27:08 Waiting for 1 join peers.  Have [node1:8088 node2:8088]. Asking nodes: [node1:8091 node2:8091 node3:8091]
```

It hang forever :( I would expect cluster to restart as soon as quorum is reached. I need to have ALL three node online to restart the cluster.

It seem to be close to #5494, shouldn't we ignore -join argument if node is already a member of a cluster ?

**Comment by PierreF** [comment]
Any updates ? It would be very useful for us to have this fixed before 0.11 release.

**Comment by pauldix** [comment]
0.11 was released today. Future versions of the open source InfluxDB will be standalone servers. See https://influxdata.com/blog/update-on-influxdb-clustering-high-availability-and-monetization/
